TictactoeVer2Tests/TictactoeTests.cs

```diff
@@ -10,61 +11,86 @@ public class TictactoeTests
         private Tictactoe Game { get; set; }
         private Mock<IOutputWriter> _mockOutputWriter;
         
-        public TictactoeTests()
+        public void InitializeTictactoeGame()
         {
             _mockOutputWriter = new Mock<IOutputWriter>();
             
             Game = new Tictactoe(_mockOutputWriter.Object);
-            Game.InitializeGame();
         }
         
         private void StartGameWith3X3Board()
         {
-            Game.InitializeGame();
-            Game.StartActualGame(3);
+            InitializeTictactoeGame();
+            
+            Game.InterpretInput("3");
+            
+            Assert.Equal(GameStatus.Playing, Game.Status);
```

Comment:
Assert the 'accepted board size' check message is displayed, rather than checking Game status directly

Reply:
Same goes for other instances of tests that check `Game.Status`